endpoints/openrtb2/auction.go

```diff
@@ -1782,7 +1782,9 @@ func (deps *endpointDeps) validateUser(req *openrtb_ext.RequestWrapper, aliases
 			return append(errL, errors.New(`request.user.ext.prebid requires a "buyeruids" property with at least one ID defined. If none exist, then request.user.ext.prebid should not be defined.`))
 		}
 		for bidderName := range prebid.BuyerUIDs {
-			if _, ok := deps.bidderMap[bidderName]; !ok {
+			normalizedCoreBidder, _ := openrtb_ext.NormalizeBidderName(bidderName)
+			coreBidder := normalizedCoreBidder.String()
+			if _, ok := deps.bidderMap[coreBidder]; !ok {
 				if _, ok := aliases[bidderName]; !ok {
 					return append(errL, fmt.Errorf("request.user.ext.%s is neither a known bidder name nor an alias in request.ext.prebid.aliases", bidderName))
 				}
```

Comment:
I think you could reuse `resolveBidder` function here. so - 
```
for bidderName := range prebid.BuyerUIDs {
       coreBidderName := resolveBidder(bidderName, aliases)
       if _, ok := deps.bidderMap[coreBidderName.String()]; !ok {
               return append(errL, fmt.Errorf("request.user.ext.%s is neither a known bidder name nor an alias in request.ext.prebid.aliases", bidderName))
       }
}
```

Since we already checked aliases so you need not to do it again. Let me know if this makes sense.

Reply:
I implemented this change locally, and all of the tests still pass. You'd need to export the `resolveBidders` function since it comes from the exchange package.

Reply:
`resolveBidder` function is in the `exchange` package and auction.go file is in the `endpoints/openrtb2/`package. Exporting the function is an unsafe option so keeping it as is.